src/wp-includes/class-wp-theme-json.php

```diff
@@ -747,8 +756,8 @@ public function __construct( $theme_json = array(), $origin = 'theme' ) {
 		$valid_block_names   = array_keys( static::get_blocks_metadata() );
 		$valid_element_names = array_keys( static::ELEMENTS );
 		$valid_variations    = static::get_valid_block_style_variations();
-		$theme_json          = static::sanitize( $this->theme_json, $valid_block_names, $valid_element_names, $valid_variations );
-		$this->theme_json    = static::maybe_opt_in_into_settings( $theme_json );
+		$this->theme_json    = static::sanitize( $this->theme_json, $valid_block_names, $valid_element_names, $valid_variations );
```

Comment:
I couldn't find anything in the [Gutenberg PR](https://github.com/WordPress/gutenberg/pull/58409/files#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7) about this change.

Any reason to set the class property `$this->theme_json` here?

If so, maybe it's worth mentioned in the annotation.

Reply:
It gets set again on the next line anyway so I wonder this is just a style/consistency thing?

Reply:
Looks like there's no functional change here. If it's not ok, let's follow-up separately.

Reply:
At one point during development I was using the `$theme_json` argument to check the theme.json version and run code conditionally based on that. Turns out the way that we merge files in `get_merged_data` (https://github.com/WordPress/wordpress-develop/pull/6616/files#diff-d6b86476eed058e7cf8b6e57fa52c4fd75b1f0907e1a9ccb0149528a24f7578bR618-R636) means that it isn't a reliable way to check if we're starting with a v2 theme.json.

I didn't switch it back because the code is functionally the same, and I think it reads better this way.